src/Components/Web.JS/package.json

```diff
@@ -16,7 +16,7 @@
   "devDependencies": {
     "@aspnet/signalr": "link:../../SignalR/clients/ts/signalr",
     "@aspnet/signalr-protocol-msgpack": "link:../../SignalR/clients/ts/signalr-protocol-msgpack",
-    "@dotnet/jsinterop": "https://dotnet.myget.org/F/aspnetcore-dev/npm/@dotnet/jsinterop/-/@dotnet/jsinterop-3.0.0-preview9.19415.3.tgz",
+    "@microsoft/dotnet-js-interop": "https://dotnet.myget.org/F/aspnetcore-dev/npm/@microsoft/dotnet-js-interop/-/@microsoft/dotnet-js-interop-3.0.0-preview9.19415.3.tgz",
```

Comment:
What keeps this URL up-to-date with the right version @pranavkm ? I don't want the dependency flows blasting this back to the old name. I didn't see anything else referring to `@dotnet/jsinterop` (I grepped the whole repo) but I may be missing something.

Reply:
It's entirely manual. We update it anytime we add update the JS package which isn't very frequent.

Reply:
Ok, so we should also update the version number when the extensions PR merges and is built.